docs/usage/python-api.rst

```diff
@@ -544,6 +544,27 @@ Currently the following parameters are supported (more will be added if useful):
     df = result.df
     print(df.head())
 
+Instead of a latlon you may alternatively use an existing station id for which to interpolate values in a manner of
+getting a more complete dataset:
+
+.. ipython:: python
+    :okwarning:
+
+    from wetterdienst.provider.dwd.observation import DwdObservationRequest
+    from wetterdienst import Parameter, Resolution
+
+    stations = DwdObservationRequest(
+        parameter=Parameter.TEMPERATURE_AIR_MEAN_200,
+        resolution=Resolution.HOURLY,
+        start_date=datetime(2022, 1, 1),
+        end_date=datetime(2022, 1, 20),
+    )
+
+    result = stations.interpolate(latlon="02480")
```

Comment:
Should we rename the parameter `latlon` or split it into two different parameters something like`latlon` and `station_id`?